diff --git a/data/de.schmidhuberj.DieBahn.metainfo.xml.in b/data/de.schmidhuberj.DieBahn.metainfo.xml.in
index 3825c94..d41ae18 100644
--- a/data/de.schmidhuberj.DieBahn.metainfo.xml.in
+++ b/data/de.schmidhuberj.DieBahn.metainfo.xml.in
@@ -15,6 +15,10 @@
   <translation type="gettext">@gettext-package@</translation>
   <update_contact>flatpak@schmiddi.anonaddy.com</update_contact>

+  <categories>
+    <category>Maps</category>
+  </categories>
+
   <url type="homepage">https://mobile.schmidhuberj.de/railway</url>
   <url type="bugtracker">https://gitlab.com/schmiddi-on-mobile/railway/-/issues</url>
   <url type="translate">https://hosted.weblate.org/engage/schmiddi-on-mobile/</url>
